nyxtools/vector.py

```diff
@@ -182,53 +182,50 @@ def prepare_move(
         shutter_time_ms: float):
 
         # Configure motion
-        yield from bps.abs_set(self.sync, 1, wait=True)
+        self.sync.put(1)
 
-        yield from bps.mv(
-            self.calc_only, True, # Check for errors first
-            self.expose, True,
-            self.hold, False,
+        self.calc_only.put(True)
+        self.expose.put(True)
+        self.hold.put(False)
 
-            self.exposure, exposure_ms,
-            self.num_samples, num_samples,
-            self.buffer_time, buffer_time_ms,
-            self.shutter_lag_time, shutter_lag_time_ms,
-            self.shutter_time, shutter_time_ms,
+        self.exposure.put(exposure_ms)
+        self.num_samples.put(num_samples)
+        self.buffer_time.put(buffer_time_ms)
+        self.shutter_lag_time.put(shutter_lag_time_ms)
+        self.shutter_time.put(shutter_time_ms)
 
-            self.o.start, o[0],
-            self.o.end, o[1],
+        self.o.start.put(o[0])
+        self.o.end.put(o[1])
 
-            self.x.start, x[0],
-            self.x.end, x[1],
+        self.x.start.put(x[0])
+        self.x.end.put(x[1])
 
-            self.y.start, y[0],
-            self.y.end, y[1],
+        self.y.start.put(y[0])
+        self.y.end.put(y[1])
 
-            self.z.start, z[0],
-            self.z.end, z[1],
+        self.z.start.put(z[0])
+        self.z.end.put(z[1])
 
-            group='vec_config'
-        )
-
-        yield from bps.wait('vec_config')
+        # Do we need to wait here?
```

Comment:
If we wait for each call to succeed, then we shouldn't need to wait at the end according to the documentation https://nsls-ii.github.io/bluesky/generated/bluesky.plan_stubs.mv.html

Reply:
Sounds good.